README.md

```diff
@@ -85,48 +85,112 @@ The strategy can be specified either on the task or as a system property for ad
 ```groovy
 gradle dependencyUpdates -Drevision=release
 ```
+ 
 
-The latest versions can be further filtered using [Component Selection Rules][component_selection_rules].
-The current version of a component can be retrieved with `currentVersion` property. For example, to
-disallow release candidates as upgradable versions from stable versions, a selection rule could be
-defined as:
+To further define which version to accept, you need to define what means an unstable version. Sadly, there are
+no agreed standard on this, but this is a good starting point:
+
+<details open>
+<summary>Groovy</summary>
 
 ```groovy
-dependencyUpdates.resolutionStrategy {
-  componentSelection { rules ->
-    rules.all { ComponentSelection selection ->
-      def isNonStable = { String version ->
-        ['alpha', 'beta', 'rc', 'cr', 'm', 'preview', 'b', 'ea'].any { qualifier ->
-          version ==~ /(?i).*[.-]$qualifier[.\d-+]*/
-        }
-      }
+def isNonStable = { String version ->
+  def stableKeyword = ['RELEASE', 'FINAL', 'GA'].any { it -> version.toUpperCase().contains(it) }
```

Comment:
I don't recall if we filter these out, but `null` version is valid. It most often happens for local file dependencies (via `flatDir`). So this could NPE.

Reply:
I would prefer that we filter out null in this case, it would be annoying in Kotlin to have to use a nullable type

Reply:
True, `null` values are annoying. However you are removing a Gradle feature by doing that, which is also undesirable.

Reply:
Obviously we could show this and assume someone using a local repository would know enough to add the null check themselves. But they might not if blindly copying samples...

Reply:
https://github.com/ben-manes/gradle-versions-plugin/pull/340/commits/925ad3524e5d65fc6706f84e5e362aefaa2d4eb2